semgrep/tests/e2e/snapshots/test_ignores/test_internal_explicit_semgrepignore/results.json

```diff
@@ -0,0 +1,205 @@
+{
```

Comment:
This is part of the reason I don't like our python tests based on those snapshots. They are very long,
and not super readable. I far prefer what we have in semgrep-core where basically the test file
contain its own assertion by having the ruleid: or MATCH: special comment, so when you look at the test code,
you also see what we're supposed to return.